python/ray/worker.py

```diff
@@ -1773,6 +1785,8 @@ def connect(node,
 
     # Register the worker with Redis.
     if mode == SCRIPT_MODE:
+        worker.current_exported_id = _random_string()
```

Comment:
I haven't figure out what exact issue the previous `_last_job_id_exported_for` fixes. 
But I guess the behavior has changed. Previously, `_last_job_id_exported_for` is compared with `current_job_id`, which can change while a worker executes tasks. 
But now, this `current_exported_id` only changes in driver.

@robertnishihara can you help confirm this?

Reply:
btw, we should prepend an underscore to a private variable name.

Reply:
> I haven't figure out what exact issue the previous `_last_job_id_exported_for` fixes.
> But I guess the behavior has changed. Previously, `_last_job_id_exported_for` is compared with `current_job_id`, which can change while a worker executes tasks.
> But now, this `current_exported_id` only changes in driver.
> 
> @robertnishihara can you help confirm this?

maybe we can add a cluster version(random str) to handle this case in worker.